kubernetes_env/kube_env.py

```diff
@@ -273,9 +279,8 @@ def deploy_filter(filter_dir):
     else:
         log.warning("Config map %s already exists!", CM_FILTER_NAME)
     # update the containers with the config map
-    cmd = f"kubectl replace -f {YAML_DIR}/bookinfo-mod-filter.yaml "
```

Comment:
Just curious - what about the bookinfo app needs to be changed?  I thought it was usable out of the box.

Reply:
You have to deploy it with the configmap in the yaml file. It can also be patched, but I have not written that sort of code yet.

Reply:
Oh interesting, thanks!